src/init.c

```diff
@@ -25,7 +25,9 @@
 #include "catalog/indexing.h"
 #include "catalog/pg_extension.h"
 #include "catalog/pg_inherits.h"
+#if PG_VERSION_NUM < 110000
```

Comment:
Bikeshedding: usually, I place conditional includes at the bottom.

Reply:
done